R/get.variable.names.r

```diff
@@ -0,0 +1,155 @@
+
+
+#' This function allows to build a data table containing the original PSID variable names (of type AB12345) for each desired year and checks
+#' on the psidonline web site whether a variable is available or not. Information is obtained on https://simba.isr.umich.edu/VS/s.aspx
+#' through the package XML.
+#'
+#' Instead of building the data table by hand and look up each variable on the online dictionary to understand in which year it is available, 
+#' it is sufficient to look at the dictionary for a given wave, select some variables of interest and type in their original name for a given 
+#' year y, as well as a list of years that one wants to be included in the panel. If the variable is not available for a given year, the 
+#' function automatically types NA in the corresponding cell.
+#'
+#' Note that, in the list of variables provided, one could include a variable from year y and a variable from year y'. This is allowed and 
+#' irrelevant to the functioning of the function. This could be useful if one wants to recover some variables that are only present in some
+#' specific waves. Only remark: if one types in the same variable for both year y and year y', it is going to show up twice in the final data 
+#' table.
+#'
+#' The following options are also available:
+#' - if scrape_name equals TRUE, then the function recovers the variable names from the online dictionary and edits them to avoid the presence
+#' of special characters such as "/", "#" or "-"; the dictionary names suggested by psidonline might be long and cryptic and are NOT always
+#' consistent over years, even if the coding of the variable is the same;
+#' - if the vector varnames is provided and scrape_name equals FALSE, then the function names the variables after the user-provided labels;
+#' varnames must be of the same length as input vector vars.
+#'
+#' Here is shown how to obtain the ind and fam data tables used in the main example function. In this example, the function is required to
+#' scrape names from psidonline.
+#'
+#' years=c(1968:1997, seq(from=1999, to=2011, by=2))
+#' i <- c("ER30046", "ER30052", "ER30066") # Variable original names for the 1970 wave
```

Comment:
i see! so you have to look up the variable names only in **one** wave, and you then go online and look in the variable description for where it says *available in waves*? If that's the case, i think it would be important to explain that above. some simple high-level explanation so that the user knows how you do this. it may not always be what the user wants - anyway, better to tell them how this works.

Reply:
I rewrote the description, hopefully it is clearer now!